src/object/obj_internal.h

```diff
@@ -139,6 +146,11 @@ struct obj_tls {
 	struct srv_profile	*ot_sp;
 };
 
+struct obj_ec_parity {
+       int             nr;
```

Comment:
structs have a prefix for fields in daos....so should probably be ep_nr (or p_nr or oep_nr)...See the definition for obj_tls above.

Also, this structure has an internal hole...Should put the larger fields first.  The compiler will put 32-bits of padding between nr and p_bufs in this case.